realsense_camera/src/base_nodelet.cpp

```diff
@@ -312,6 +312,8 @@ namespace realsense_camera
         checkError();
         detected_camera_msg = detected_camera_msg + ", Motion Module FW: " + motion_module_fw;
       }
+
+      checkFirmwareValidation("validatedFirmware", rs_detected_device, "typeOfFirmware");
```

Comment:
If you re-order the arguments and use default arguments in the declaration, you don't have to pass these dummy strings. See this: http://en.cppreference.com/w/cpp/language/default_arguments